cmd/gossamer/account.go

```diff
@@ -49,12 +49,14 @@ func handleAccounts(ctx *cli.Context) error {
 		log.Info("generating keypair...")
 
 		// check if --ed25519 or --sr25519 is set
-		keytype := utils.Sr25519KeyType
+		keytype := crypto.Sr25519Type
 		if flagtype := ctx.Bool(utils.Sr25519Flag.Name); flagtype {
-			keytype = utils.Sr25519KeyType
+			keytype = crypto.Sr25519Type
 		} else if flagtype := ctx.Bool(utils.Ed25519Flag.Name); flagtype {
-			keytype = utils.Ed25519KeyType
-		}
+			keytype = crypto.Ed25519Type
+		} /*else if flagtype := ctx.Bool(utils.Secp256k1Flag.Name); flagtype {
+			keytype = crypto.Secp256k1Type
+		}*/
```

Comment:
Are we waiting to add secp?

Reply:
yeah, I can uncomment this when my other PR gets merged

Reply:
PR merged